### PR TITLE
Rustfmt

### PR DESCRIPTION
**Pull request by FredTheDino**

### CODE REVIEW

File: sylt-common/src/block.rs [hunk 1]
```diff
@@ -87,10 +87,9 @@ impl Block {
                 i.red(),
                 s,
                 match (s, constants) {
-                    (Op::Constant(c), Some(constants))
-                    | (Op::Link(c), Some(constants))
-                      => format!("    => {:?}", &constants[*c]),
-                    _ => "".to_string()
+                    (Op::Constant(c), Some(constants)) | (Op::Link(c), Some(constants)) =>
+                        format!("    => {:?}", &constants[*c]),
+                    _ => "".to_string(),
```
Comment (FredTheDino) [inline sylt-common/src/block.rs:92]: Disagree.

File: sylt-common/src/error.rs [hunk 2]
```diff
@@ -115,15 +113,23 @@ impl fmt::Display for Error {
...
 
         match self {
-            Error::RuntimeError { kind, phase, file, line, message } => {
+            Error::RuntimeError {
+                kind,
+                phase,
+                file,
+                line,
+                message,
+            } => {
```
Comment (FredTheDino) [inline sylt-common/src/error.rs:122]: Disagree.

File: sylt-common/src/value.rs [hunk 3]
```diff
@@ -157,20 +161,26 @@ pub enum MatchableValue<'t> {
...
     match value {
         Tuple(inner) => {
-            match (inner.get(0), inner.get(1), inner.get(2), inner.get(3), inner.get(4)) {
+            match (
+                inner.get(0),
+                inner.get(1),
+                inner.get(2),
+                inner.get(3),
+                inner.get(4),
+            ) {
```
Comment (FredTheDino) [inline sylt-common/src/value.rs:175]: Disagree.

File: sylt-parser/src/lib.rs [hunk 4]
```diff
@@ -165,7 +165,6 @@ pub enum StatementKind {
     },
 
     // TODO(ed): break and continue
-
```
Comment (FredTheDino) [inline sylt-parser/src/lib.rs:168]: Disagree.

File: sylt-parser/src/lib.rs [hunk 5]
```diff
@@ -1362,7 +1366,7 @@ mod expression {
                     let (_ctx, expr) = expression(ctx)?;
                     exprs.push(expr);
                     ctx = _ctx; // assign to outer
-                    // Not a tuple, until it is.
+                                // Not a tuple, until it is.
```
Comment (FredTheDino) [inline sylt-parser/src/lib.rs:1369]: ?

File: sylt-parser/src/lib.rs [hunk 6]
```diff
@@ -1561,11 +1565,7 @@ fn outer_statement<'t>(ctx: Context<'t>) -> ParseResult<Statement> {
     let (ctx, stmt) = statement(ctx)?;
     use StatementKind::*;
     match stmt.kind {
-        Blob { ..}
-        | Definition { .. }
-        | Use { .. }
-        | EmptyStatement
-        => Ok((ctx, stmt)),
+        Blob { .. } | Definition { .. } | Use { .. } | EmptyStatement => Ok((ctx, stmt)),
```
Comment (FredTheDino) [inline sylt-parser/src/lib.rs:1568]: Disagree.

File: sylt-std/src/lingon.rs [hunk 7]
```diff
@@ -121,7 +133,17 @@ sylt_macro::extern_function!(
...
+    x: &f64,
+    y: &f64,
+    w: &f64,
+    h: &f64,
+    rot: &f64,
+    r: &f64,
+    g: &f64,
+    b: &f64,
+    a: &f64,
+) {
```
Comment (FredTheDino) [inline sylt-std/src/lingon.rs:146]: Disagree.

File: sylt-std/src/lingon.rs [hunk 8]
```diff
@@ -210,7 +232,20 @@ sylt_macro::extern_function!(
...
+    w: &f64,
+    h: &f64,
+    gx: &i64,
+    gy: &i64,
+    rot: &f64,
+    r: &f64,
+    g: &f64,
+    b: &f64,
+    a: &f64,
+) -> Value {
```
Comment (FredTheDino) [inline sylt-std/src/lingon.rs:248]: Disagree.

File: sylt-typechecker/src/lib.rs [hunk 9]
```diff
@@ -112,21 +111,22 @@ fn typecheck_block(block_slot: usize, prog: &Prog, global_types: Vec<Type>, verb
...
     pub(crate) fn new(block: &Rc<RefCell<Block>>, global_types: Vec<Type>) -> Self {
         let mut vm = Self {
-            upvalues: block.borrow().upvalues.iter().map(|(_, _, ty)| ty).cloned().collect(),
+            upvalues: block
+                .borrow()
+                .upvalues
+                .iter()
+                .map(|(_, _, ty)| ty)
+                .cloned()
+                .collect(),
```
Comment (FredTheDino) [inline sylt-typechecker/src/lib.rs:129]: On the fence.

File: sylt-typechecker/src/lib.rs [hunk 10]
```diff
@@ -142,7 +142,7 @@ impl VM {
     }
 
     fn print_stack(&self) {
-        print!("        [", );
+        print!("        [",);
```
Comment (FredTheDino) [inline sylt-typechecker/src/lib.rs:145]:
```suggestion
        print!("        [");
```
?

File: sylt/src/bin/time.rs [hunk 11]
```diff
@@ -7,10 +7,18 @@ pub struct Args {
     #[options(free, required, help = "The file to run")]
     pub run_file: PathBuf,
 
-    #[options(short = "r", long = "runs", help = "If set, how many times the program should be run at most")]
+    #[options(
+        short = "r",
+        long = "runs",
+        help = "If set, how many times the program should be run at most"
+    )]
```
Comment (FredTheDino) [inline sylt/src/bin/time.rs:14]: <3

File: sylt_macro/src/lib.rs [hunk 12]
```diff
@@ -70,31 +73,49 @@ pub fn extern_function(tokens: proc_macro::TokenStream) -> proc_macro::TokenStre
...
-    let matches: Vec<_> = parsed.blocks.iter()
-        .map(|ExternBlock { pattern, return_ty, ..}| quote! { #pattern #return_ty })
+    let matches: Vec<_> = parsed
+        .blocks
+        .iter()
+        .map(
+            |ExternBlock {
+                 pattern, return_ty, ..
+             }| quote! { #pattern #return_ty },
+        )
```
Comment (FredTheDino) [inline sylt_macro/src/lib.rs:91]: Disagree.

**Review by FredTheDino — COMMENTED** [review]
I agree with most of the changes. Some minor nits though.

**Comment by FredTheDino** [comment]
@sornas are we doing this or no?

**Comment by sornas** [comment]
Yeah I'm looking over it now.

**Comment by sornas** [comment]
Oh wow the merge conflicts suckk, I can do a new pass.

**Comment by sornas** [comment]
Merged (sort-of) in #256

**Pull request closed without merging**